docs/developers/contracts/technical-references/tick-ratio.md

```diff
@@ -1,100 +1,96 @@
 ---
-description: Introduction to the concepts of tick and ratio
+description: Introduction to the concepts of ticks, ratios, and prices
 sidebar_position: 1
 ---
 
-# Ticks and ratio
+# Ticks, ratios, and prices
 
-On Mangrove, markets (and therefore [offer lists](./taking-and-making-offers/offer-list.md)) are discretized in ticks. That means that every offer posted and every order placed are using them. Since this is of importance to use Mangrove as a developer, we will spend time in this page explaining what they are and how they are connected to pricing ([ratio](#ratio)).
+On Mangrove, the price space of a market is discretized into _ticks_: It contains the price 1 `quote/base` and the smallest price increment (and decrement) is 1 basis point = 0.01%.
```

Comment:
Consider referencing "beware decimals" since this only holds for equal amount of decimals. For unequal amounts of decimals, 1 is most likely not representable.

Reply:
Addressed in https://github.com/mangrovedao/mangrove-docs/pull/203